t/base/protocols/file.t

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
+use utf8;
+use strict;
+use warnings;
+use Test::More tests => 4;
+
+use File::Basename qw(dirname);
+use LWP::Simple qw(get);
+use File::Temp qw(tempfile);
+
+require POSIX;
+POSIX::setlocale(&POSIX::LC_ALL, 'ja-JP.UTF-8');
```

Comment:
I asked on irc.perl.org about `setlocale` and the feedback I got was:

> on linux, if the given locale isn't available it won't work

Having said that, this test passes locally on my Mac with those lines removed.

Reply:
Also:

> also it should return the new locale if properly set
11:01 AM 
so possibly it should skip if setting the locale fails
11:02 AM 
but also, for the purposes of that test, the collation doesn't matter, so it may be better to use a locale that is more common, like en_US.UTF-8
11:02 AM 
the language part is irrelevant
11:02 AM 
as long as it exists